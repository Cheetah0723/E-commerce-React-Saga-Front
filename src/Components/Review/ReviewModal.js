

import React from 'react';
import { makeStyles } from '@material-ui/core/styles';
import Modal from '@material-ui/core/Modal';
import ReviewForm from "./ReviewForm"
import "./ReviewModal.css"

const useStyles = makeStyles((theme) => ({
    modal: {
        backgroundColor: theme.palette.background.paper,
        paddingTop:"80px",
        maxWidth: "1400px",
        overflow: "scroll",
        alignItems: 'center',
        justifyContent: 'center',
        width:"600px",
        marginLeft:"30%"
    },
}));

export default function ReviewModal({ productName, openStatus, handleClose, productId }) {
    const classes = useStyles();
    return (
        <Modal
            aria-labelledby="transition-modal-title"
            aria-describedby="transition-modal-description"
            className={classes.modal}
            open={openStatus}
            onClose={handleClose}
        >
            <div className={classes.paper}>
                <ReviewForm productName={productName} productId={productId}
                    initialState=""
                />
            </div>
        </Modal>
    );
}




