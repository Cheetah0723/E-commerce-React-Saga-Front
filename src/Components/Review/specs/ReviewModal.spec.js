import React from 'react'
import ReviewModal from "../ReviewModal"
import renderer from 'react-test-renderer'

describe("ReviewModal", () => {

    it('renders correctly', () => {
        const test = (arg) => { console.log(arg) }
        const component = renderer.create(<ReviewModal productName="test"
            openStatus={false} handleClose={test} />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
})