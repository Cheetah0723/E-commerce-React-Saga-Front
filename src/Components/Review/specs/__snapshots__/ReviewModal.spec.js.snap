// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewModal renders correctly 1`] = `null`;
